cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -161,7 +180,8 @@ public void execute() {
                         do {
                             out.print("Module for the service: (no module) ");
                             packageName = scanner.nextLine().trim();
-                        } while (!validatePkgName(projectPath, packageName));
+                        }
+                        while (!validatePkgName(projectPath, packageName));
```

Comment:
Better to revert this change. Can be applied to few `while` loops which has been modified below as well.